### DISCUSSION TITLE
geopandas and dask based redridding

### DISCUSSION THREAD

**Discussion by jmccreight** [hytest-org/hytest#307] [Data Processing and Analysis]
@rsignell-usgs You mentioned a geopandas based regridding approach that has a dask back end. I was hoping you could share a link to that here.

**Comment by rsignell-usgs** [comment]
@jmccreight thanks for reminding me in an open setting! 

Here's the [original concept by Ryan in the Pangeo Discourse Forum](https://discourse.pangeo.io/t/conservative-region-aggregation-with-xarray-geopandas-and-sparse/2715), and [my contribution](https://discourse.pangeo.io/t/conservative-region-aggregation-with-xarray-geopandas-and-sparse/2715/16) a bit further along in the discussion.

Also relevant: in the post/notebook he discusses that to do the regridding, you should be able to just the regridding by multiply the sparse weight matrix by the original data using:
```
regridded = xr.dot(ds.precip, weights_sparse)
```
but that it doesn't work because `sparse` doesn't implement `einsum`. 

@dcherian [figured that part out later in the discussion](https://discourse.pangeo.io/t/conservative-region-aggregation-with-xarray-geopandas-and-sparse/2715/30)!

@alaws-USGS had a nice tutorial using this approach somewhere in the HyTEST training -- hopefully he,  @amsnyder, @sfoks or someone else from they HyTEST team can chime in with the location of that!

**Comment by rmcd-mscb** [comment]
@jmccreight There is also [gdptools](https://gdptools.readthedocs.io/en/develop/)
